### PR TITLE
add robot benchmark

### PR DESCRIPTION
**Pull request by baggepinnen**
Requires
- New release of MTK
- https://github.com/JuliaComputing/Multibody.jl/pull/61
- https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/296
- New release of MTKstdlib

### CODE REVIEW

**Comment by baggepinnen** [comment]
First run 
image


second run
image


Julia timings during second run is split up like this
image

File: benchmarks/ModelingToolkit/Multibody_Robot.jmd [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+## OpenModelica
+
+```julia
+cd(@__DIR__)
+using Pkg
+Pkg.activate(".")
+using OMJulia
+mod = OMJulia.OMCSession();
+OMJulia.sendExpression(mod, "getVersion()")
```
Comment (ChrisRackauckas) [inline benchmarks/ModelingToolkit/Multibody_Robot.jmd:77]:
```suggestion
mod = OMJulia.OMCSession();
OMJulia.sendExpression(mod, "getVersion()")
OMJulia.sendExpression(mod, "installPackage(Modelica)")
```

**Comment by avinashresearch1** [comment]
Dymola results: 

Run Number | Translation time | Total time
1, 5.075, 5.267

2, 3.912, 4.112

3, 4.024, 4.255

**Comment by ChrisRackauckas** [comment]
Multibody isn't in the registry?

**Comment by baggepinnen** [comment]
No, no release has been made yet

**Pull request merged**